### PR TITLE
Lightbox split

### PR DESCRIPTION
**Pull request by uplne**
Another one. Lightbox last change was 9 weeks ago. Because users are mainly visiting content pages this won't have as big impact as splitting football but it's still 3.9KB (gzipped) saved on facia and possible other pages.

CC @michaelwmcnamara @Jholder112233 
- [x] - works on article pages
- [x] - works on gallery pages
- [x] - works on liveblog pages
- [x] - works on eye witness and cartoons pages (image content)

### CODE REVIEW

File: common/app/views/fragments/javaScriptLaterSteps.scala.html [hunk 1]
```diff
@@ -14,24 +14,25 @@
             @curlPaths.map { case (module, path) =>
                 '@module': '@Static(path)',
             }
-            core:                      '@Static("javascripts/core.js")',
```
Comment (robertberry) [inline common/app/views/fragments/javaScriptLaterSteps.scala.html]: Why's Git think all these lines have changed? :confused:

**Comment by robertberry** [comment]
Cool :+1:

**Comment by robertberry** [comment]
Every little helps btw. It's not just download size, but also parse time.

File: common/app/views/fragments/javaScriptLaterSteps.scala.html (hunk 1 above)
Comment (uplne) [inline common/app/views/fragments/javaScriptLaterSteps.scala.html]: 'bootstraps/image-content': is the new longest line so I had to add one space to all lines :smile:

**Comment by michaelwmcnamara** [comment]
One weird thing - looking at http://localhost:9000/world/picture/2015/feb/24/eyewitness-oscars-2015-hollywood-usa
when I expand the image to view the fullsize one (in the lightbox) it works ok, but when I go to close this lightbox, it takes 2 clicks to do so.

**Comment by uplne** [comment]
@michaelwmcnamara nice catch. I've noticed that already too. It's doing it for eyewitness and cartoons. I am trying to figure out why.

**Comment by michaelwmcnamara** [comment]
:+1:  Looks good!

**Pull request merged**